src/sentry/snuba/discover.py

```diff
@@ -940,16 +848,23 @@ def top_events_timeseries(
                 # A user field can be any of its field aliases, do an OR across all the user fields
                 elif field == "user":
                     snuba_filter.conditions.append(
-                        [[resolve_column(user_field), "IN", values] for user_field in user_fields]
+                        [
+                            [resolve_column(Dataset.Discover)(user_field), "IN", values]
```

Comment:
Would we be able to make a resolver around line 829 and re-use it multiple times?